tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "forceConsistentCasingInFileNames": true,
     "declaration": true,
     "declarationMap": true,
-    "sourceMap": true
+    "sourceMap": true,
+    "resolveJsonModule": true,
```

Comment:
he reason I added this property is so in AnalyticsEventReporter I can import values from package.json:

import { name, version } from '../package.json';

It complains without this property in the config.

Importing from package.json makes setting clientSdk super clean.